src/libraries/System.Private.CoreLib/src/System/IO/FileInfo.cs

```diff
@@ -122,11 +142,22 @@ public FileStream Open(FileMode mode, FileAccess access, FileShare share)
             => new FileStream(NormalizedPath, mode, access, share);
 
         public FileStream OpenRead()
-            => new FileStream(NormalizedPath, FileMode.Open, FileAccess.Read, FileShare.Read, 4096, false);
+            => new FileStream(NormalizedPath, FileMode.Open, FileAccess.Read, FileShare.Read, File.DefaultBufferSize, false);
 
         public FileStream OpenWrite()
             => new FileStream(NormalizedPath, FileMode.OpenOrCreate, FileAccess.Write, FileShare.None);
 
+#if NET6_0_OR_GREATER
```

Comment:
Similarly to `File` new methods, please move them to `FileInfo.netcoreapp.cs` file